runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -998,6 +999,8 @@ void chpl_comm_init(int *argc_p, char ***argv_p) {
   envUseAmTxCntr = chpl_env_rt_get_bool("COMM_OFI_AM_TX_COUNTER", false);
   envUseAmRxCntr = chpl_env_rt_get_bool("COMM_OFI_AM_RX_COUNTER", false);
   envProgressCntr = chpl_env_rt_get_bool("COMM_OFI_PROGRESS_COUNTER", false);
+
+  envUseInject = chpl_env_rt_get_bool("COMM_OFI_INJECT", true);
```

Comment:
Thoughts on splitting this into:
 - `CHPL_RT_COMM_OFI_INJECT_RMA`
 - `CHPL_RT_COMM_OFI_INJECT_AMO`
 - `CHPL_RT_COMM_OFI_INJECT_AM`

This would allow us to control our main types of comm separately (where I'm imagining for 1.26 we may disable inject for RMA/AMO, but not for AM)

(and maybe in a separate PR changing the cntr env vars to use the same three categories?)

Reply:
Splitting into three environment variables can be done. As aside, we probably should be documenting all these runtime environment variables somewhere. 

I'm not sure about separating the counter environment variables similarly. Maybe open an issue in which we can hash out what that would look like.

Reply:
> Splitting into three environment variables can be done. As aside, we probably should be documenting all these runtime environment variables somewhere.

For now I expect these to be developer option only, so I don't feel strongly about documenting them. If/when they become user-facing options I'm in favor of documenting.